check_opnsense.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Move this change into separate commit (eg "Change python interpreter to python3 explicitly")